src/main/java/com/treasuredata/bigdam/log/Log.java

```diff
@@ -344,6 +342,21 @@ else if (attributeKeysMasked.contains(key)) {
         return event;
     }
 
+    private Map<String, Object> buildEvent(final String messageKey, final String message, final Throwable e, final Map<String, ? extends Object> attrs)
+    {
+        HashMap<String, Object> event = new HashMap<>();
+        if (messageKey != null) {
+            event.put(messageKey, message);
+        }
+        if (e != null) {
+            event.put("errorClass", e.getClass().getName());
+            event.put("error", e.getMessage());
```

Comment:
Ah, I noticed the cause of this exception won't be sent. How about sending a cause recursively if it exists?

Reply:
Do you mean to add `causeClass` and `cause` fields?

Reply:
Hmm, it's not recursive way. How can do it?

Reply:
> Do you mean to add causeClass and cause fields?
Right. Also, `List` can represent nested multiple causes, I think.

However, I thought this information would be sent to Sentry. So adding cause exceptions here seemed great. According to your earlier comment, it's not sent to Sentry and maybe we don't need to do that.

Reply:
I see. I agree that we can refer Sentry dashboard when we want to investigate about errors in details. I'll skip cause exceptions for now.